src/frontend/sharedComponents/DeviceCard.tsx

```diff
@@ -44,30 +54,60 @@ export const DeviceCard = ({
       throw new ExhaustivenessError(deviceConnectionStatus);
   }
 
+  const {navigate} = useNavigationFromRoot();
+  const {formatMessage} = useIntl();
+
   return (
     <TouchableOpacity
       disabled={!onPress || isDisconnected}
       onPress={() => (onPress ? onPress() : {})}
-      style={[styles.container, style]}>
-      <DeviceNameWithIcon
-        name={name}
-        deviceConnectionStatus={deviceConnectionStatus}
-        thisDevice={thisDevice}
-        deviceType={deviceType}
-        deviceId={deviceId}
-        iconSize={75}
-      />
+      style={[style]}>
+      <View style={styles.container}>
+        <DeviceNameWithIcon
+          style={{flexShrink: 1}}
+          name={name}
+          deviceConnectionStatus={deviceConnectionStatus}
+          thisDevice={thisDevice}
+          deviceType={deviceType}
+          deviceId={deviceId}
+          iconSize={75}
+        />
+        <View style={{alignSelf: 'flex-start'}}>
+          {dateAdded && (
+            <Text style={{fontSize: 12, marginBottom: 10}}>
```

Comment:
based on design, this should be a lighter grey color:

<img width="200" alt="image" src="https://github.com/user-attachments/assets/ed0487c4-d07e-497f-8b1b-c3980edcf758">

Reply:
https://github.com/digidem/comapeo-mobile/pull/674/commits/7704b2a2aa999a780a5aa0c368dc5b197b353e89